src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -39,6 +52,26 @@ export default function ProfilePage(): JSX.Element {
         return <ChangePasswordForm />;
     };
 
+    const handleToggleChange = () => {
+        if (userRecord) {
+            updateUserTimeFormatOption(!is24Hour, userRecord?.uid);
+        }
+        setIs24Hour(!is24Hour);
+    };
+
+    const ShowChangeDisplayForm = (): JSX.Element => {
+        return (
+            <div id="toggleContent">
+                <Toggle
+                    defaultChecked={is24Hour}
+                    icons={false}
+                    onChange={handleToggleChange}
+                />
+                <p>{is24Hour ? 'Use 24-Hour Clock' : 'Use 12-Hour Clock'}</p>
```

Comment:
I think it would be better if the text for military time or 12 hour was fixed. The toggle should indicate whether one setting is either on or off.

Reply:
Okay, Amy said the same thing. We can definitely change that.